Dockerfile

```diff
@@ -13,7 +13,9 @@ FROM base AS build
 RUN apk update && apk --no-cache add \
   python3 \
   make \
-  g++
+  g++ \
+  py3-setuptools \
+  py3-distutils-extra  # Add distutils for Python
```

Comment:
Thank you for your contribution :)
Could you clarify the reason for adding `py3-distutils-extra`? It seems that the build works fine with just `py3-setuptools`.